src/clj/puppetlabs/services/jruby_pool_manager/impl/jruby_internal.clj

```diff
@@ -42,18 +49,39 @@
     :force RubyInstanceConfig$CompileMode/FORCE
     :off RubyInstanceConfig$CompileMode/OFF))
 
+(schema/defn ^:always-validate get-profiling-mode :- RubyInstanceConfig$ProfilingMode
+  [config-profiling-mode :- jruby-schemas/SupportedJRubyProfilingModes]
+  (RubyInstanceConfig$ProfilingMode/valueOf (cstring/upper-case (name config-profiling-mode))))
```

Comment:
cursive was very unhappy about this line but it works fine?

Reply:
looks fine, what's wrong with it

Reply:
incorrect arity of 1 for `RubyInstanceConfig$ProfilingMode/valueOf`